user-interface/src/lib/hooks/UseApi.ts

```diff
@@ -20,7 +19,8 @@ let context: ApiClient;
  * @returns ApiClient
  */
 function legacyConfiguration(): ApiClient {
-  return import.meta.env['CAMS_PA11Y'] === 'true' ? MockApi : Api;
+  const api = import.meta.env['CAMS_PA11Y'] === 'true' ? MockApi : Api;
```

Comment:
We can revert this. I believe this got in here because I wanted to set a breakpoint an inspect the "api" instance in use.